js/src/views/Account/Transactions/Transaction/transaction.js

```diff
@@ -160,7 +161,14 @@ export default class Transaction extends Component {
     const { api } = this.context;
     const ether = api.util.fromWei(value);
 
-    return `${ether.toFormat(5)}`;
+    if (ether.eq(0)) {
+      return { value: null, token: null };
+    }
+
+    return {
+      value: `${ether.toFormat(5)}`,
+      token: <small>ΞTH</small>
```

Comment:
Agreed. As commented there, we need a refactor of these completely - I picked up that habit from the Signer code. In addition, we actually are repeating these things everywhere (not just here), really not a good DRY approach.